keras/ops/core.py

```diff
@@ -629,3 +630,19 @@ def vectorized_map(function, elements)
     a single list of tensor arguments.
     """
     return backend.core.vectorized_map(function, elements)
+
+
+@keras_export("keras.ops.is_tensor")
+def is_tensor(x):
+    """Check whether the given object is a tensor.
+
+    Note: This checks for backend specific tensors so passing a tensorflow
+    tensor would return False if your backend is pytorch or jax.
```

Comment:
`False` (backticks)

PyTorch
JAX

Reply:
TIL JAX used to stand for ""Just After eXecution" https://github.com/google/jax/discussions/9019